uint8 n_experiments
uint8 i_current_experiment
uint8 n_actions
uint8 i_current_action
string[] action_names
uint8 n_steps
uint8[] frame_types
uint8 i_current_step
uint8[] r_gripper_states
uint8[] l_gripper_states
string[] r_ref_frames
string[] l_ref_frames
pr2_pbd_interaction/Object[] objects
